components/input-controls/range-control/index.js

```diff
@@ -1,13 +1,15 @@
 /**
  * WordPress dependencies
  */
-import { Dashicon, withInstanceId, Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import BaseControl from './../base-control';
+import Button from '../../button';
```

Comment:
It's a shame this doesn't work, but it's frustrated me in the past in my attempts to do so:

```js
import { Dashicon, withInstanceId, Button } from '../../';
```

Reply:
Yes, this syntax would be better :(